### PR TITLE
Sign in page

### PR DESCRIPTION
**Pull request by Melichka**
image
image

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -23,6 +24,7 @@ function App() {
           <Routes>
             <Route path="/" Component={SearchPage} />
             <Route path="/city" Component={CityPage} />
+            <Route path="/signIn" Component={SignInPage} />
```
Comment (Uzhastin-Nikita) [inline frontend/src/App.tsx:27]: Do you think maybe we should use a `kebab-case` for the Routes?

File: frontend/src/components/AuthorizationForm/AuthorizationForm.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { Children } from "react";
+import { PropsWithChildren } from "react";
+import { SignInSwitcher } from "../SignInSwitcher";
+import { SignUpSwitcher } from "../SignUpSwitcher";
+import { LogIn } from "../LogIn";
+import { Password } from "../Password";
+import { SubmitButton } from "../SubmitButton";
+import { Typography } from "@mui/material";
+import { VARIANTS } from "../../pages/signIn/constants";
+import styles from "./styles.module.scss";
```
Comment (Uzhastin-Nikita) [inline frontend/src/components/AuthorizationForm/AuthorizationForm.tsx:10]: Rewrite relative imports to absolute imports, please

File: frontend/src/components/NavigateButton/NavigateButton.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  isSwitcher,
+  isMain,
+  onClick
+}) => {
+  return (
+    <Button
+      className={classNames({
+        [styles["activSwitcher"]]: isActiveSwitcher,
+        [styles["switcher"]]: isSwitcher,
+        [styles["default"]]: !isSwitcher && !isMain && !isActiveSwitcher,
```
Comment (Uzhastin-Nikita) [inline frontend/src/components/NavigateButton/NavigateButton.tsx:31]: I think, that we can create separate variable for `!isSwitcher && !isMain && !isActiveSwitcher`

Something like this: 

```
const isDefault = !isSwitcher && !isMain && !isActiveSwitcher
        

[styles["default"]]: isDefault 


```

File: frontend/src/components/Title/styles.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 60%;
+  cursor: pointer;
+}
+
+.upper-title {
+  display: flex;
+  color: #8B003F;
```
Comment (Uzhastin-Nikita) [inline frontend/src/components/Title/styles.css:11]: Use scss-mixin, please

File: frontend/src/components/NavigateButton/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+.main {
+  display: flex;
+  border-color: $white-border;
+}
+
+.main:hover {
+  display: flex;
+  border-color: $activ-button;
+}
```
Comment (Uzhastin-Nikita) [inline frontend/src/components/NavigateButton/styles.module.scss:41]: Please, could you use SCSS selectors

https://sass-lang.com/documentation/style-rules/parent-selector/

```suggestion
.main {
  display: flex;
  border-color: $white-border;

  &:hover {
    display: flex;
    border-color: $activ-button;
}
```

**Review by Uzhastin-Nikita — APPROVED** [review]

File: frontend/public/assets/images/bubbleforsigninpage.svg
Comment (the-homeless-god) [inline frontend/public/assets/images/bubbleforsigninpage.svg:1]: bubble-for-sign-in-page.svg

File: frontend/src/App.tsx [hunk 6]
```diff
@@ -6,15 +6,18 @@ import StyledEngineProvider from "@mui/material/StyledEngineProvider";
 
 import { CityPage } from "src/pages/city";
 import { SearchPage } from "src/pages/search";
+import { SignInPage } from "src/pages/signIn";
 
-import { useTranslation, Trans } from 'react-i18next';
+import { Style } from "./pages/style/components";
+
+import { useTranslation, Trans } from "react-i18next";
```
Comment (the-homeless-god) [inline frontend/src/App.tsx:13]: Sort imports

File: frontend/src/components/LoginForm/LoginForm.tsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { LogIn } from "../LogIn/LogIn";
+import { Password } from "../Password";
+
```
Comment (the-homeless-god) [inline frontend/src/components/LoginForm/LoginForm.tsx:5]:
```suggestion
import { LogIn } from "src/components/LogIn";
import { Password } from "src/components/Password";

```

File: frontend/src/components/RegisterForm/RegisterForm.tsx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { LogIn } from "../LogIn/LogIn";
+import { Password } from "../Password";
```
Comment (the-homeless-god) [inline frontend/src/components/RegisterForm/RegisterForm.tsx:4]: Same

File: frontend/src/components/SignInButton/SignInButton.tsx [hunk 9]
```diff
@@ -1,17 +1,17 @@
 import React from "react";
 
-import { Button, Typography } from "@mui/material";
+import { Link } from "react-router-dom";
+
+import { NavigateButton } from "../NavigateButton";
```
Comment (the-homeless-god) [inline frontend/src/components/SignInButton/SignInButton.tsx:5]: Same

File: frontend/src/components/SignInSwitcher/SignInSwitcher.tsx [hunk 10]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import { NavigateButton } from "../NavigateButton";
```
Comment (the-homeless-god) [inline frontend/src/components/SignInSwitcher/SignInSwitcher.tsx:3]: Same

File: frontend/src/components/SignUpSwitcher/SignUpSwitcher.tsx [hunk 11]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+import { NavigateButton } from "../NavigateButton";
```
Comment (the-homeless-god) [inline frontend/src/components/SignUpSwitcher/SignUpSwitcher.tsx:3]: Same

File: frontend/src/components/SupportButton/SupportButton.tsx [hunk 12]
```diff
@@ -1,15 +1,9 @@
 import React from "react";
 
-import { Button, Typography } from "@mui/material";
-
 import { SUPPORT } from "./constants";
 
-import styles from "./styles.module.scss";
+import { NavigateButton } from "../NavigateButton";
```
Comment (the-homeless-god) [inline frontend/src/components/SupportButton/SupportButton.tsx:5]: Same

File: frontend/src/components/Title/Title.tsx [hunk 13]
```diff
@@ -1,10 +1,14 @@
 import React from "react";
 
+import { useHomeNavigation } from "../../hooks/useHomeNavigation";
```
Comment (the-homeless-god) [inline frontend/src/components/Title/Title.tsx:3]: Same

File: frontend/src/components/styles/variables/_color.scss [hunk 14]
```diff
@@ -6,3 +6,5 @@ $bottom-title-color: #A65400;
 $text-color: #a61700;
 $default-button-color: #FFBA73;
 $white-border: #fff;
+$modal-bg: rgba(255, 134, 115, 0.7 );
+$activ-button: #FFA040;
```
Comment (the-homeless-god) [inline frontend/src/components/styles/variables/_color.scss:10]:
```suggestion
$active-button: #FFA040;
```

**Review by the-homeless-god — APPROVED** [review]
Let's fix all comments as follow up, take it to the technical debt. Rarely done 🥇

**Pull request merged**